automation/taskcluster/lib/util.py

```diff
@@ -0,0 +1,27 @@
+import json
+import re
+
+
+def convert_camel_case_into_kebab_case(string):
+    # Inspired from https://stackoverflow.com/questions/1175208/elegant-python-function-to-convert-camelcase-to-snake-case  # noqa: E501
+    first_pass = re.sub('(.)([A-Z][a-z]+)', r'\1-\2', string)
+    return re.sub('([a-z0-9])([A-Z])', r'\1-\2', first_pass).lower()
+
+
+def populate_chain_of_trust_required_but_unused_files():
```

Comment:
I'm allergic to `util` files, is there a way we can name it to something more specific, or move the functions?
For example, `decision_task.py` is the only file that uses these, perhaps these functions can be moved there?
Alternatively, perhaps we can rename this file to something like `chainoftrust.py`?

Reply:
Moved these functions into `lib.chain_of_trust.py`